Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Render/MarkdownRenderer.Inlines.cs

```diff
@@ -0,0 +1,493 @@
+                FontFamily = EmojiFontFamily ?? DefaultEmojiFont,
+                Text = element.Text
+            };
+
+            inlineCollection.Add(emoji);
+        }
+
+        /// <summary>
+        /// Renders a text run element.
+        /// </summary>
+        /// <param name="element"> The parsed inline element to render. </param>
+        /// <param name="context"> Persistent state. </param>
+        protected override void RenderTextRun(TextRunInline element, IRenderContext context)
+        {
+            InternalRenderTextRun(element, context);
+        }
+
+        private Run InternalRenderTextRun(TextRunInline element, IRenderContext context)
+        {
+            var localContext = context as InlineRenderContext;
+            if (localContext == null)
+            {
+                throw new RenderContextIncorrectException();
+            }
+
+            var inlineCollection = localContext.InlineCollection;
+
+            // Create the text run
+            Run textRun = new Run
+            {
+                Text = CollapseWhitespace(context, element.Text)
+            };
+
+            // Add it
+            inlineCollection.Add(textRun);
+            return textRun;
+        }
+
+        /// <summary>
+        /// Renders a bold run element.
+        /// </summary>
+        /// <param name="element"> The parsed inline element to render. </param>
+        /// <param name="context"> Persistent state. </param>
+        protected override void RenderBoldRun(BoldTextInline element, IRenderContext context)
+        {
+            var localContext = context as InlineRenderContext;
+            if (localContext == null)
+            {
+                throw new RenderContextIncorrectException();
+            }
+
+            // Create the text run
+            Span boldSpan = new Span
+            {
+                FontWeight = FontWeights.Bold
+            };
+
+            var childContext = new InlineRenderContext(boldSpan.Inlines, context)
+            {
+                Parent = boldSpan,
+                WithinBold = true
+            };
+
+            // Render the children into the bold inline.
+            RenderInlineChildren(element.Inlines, childContext);
+
+            // Add it to the current inlines
+            localContext.InlineCollection.Add(boldSpan);
+        }
+
+        /// <summary>
+        /// Renders a link element
+        /// </summary>
+        /// <param name="element"> The parsed inline element to render. </param>
+        /// <param name="context"> Persistent state. </param>
+        protected override void RenderMarkdownLink(MarkdownLinkInline element, IRenderContext context)
+        {
+            var localContext = context as InlineRenderContext;
+            if (localContext == null)
+            {
+                throw new RenderContextIncorrectException();
+            }
+
+            // HACK: Superscript is not allowed within a hyperlink.  But if we switch it around, so
```

Comment:
@nmetulev is this a platform issue?  Are we tracking it somewhere?

Reply:
This is a long standing issue I believe.